pio/src/iSpindel.cpp

```diff
@@ -75,25 +75,25 @@ uint32_t DSreqTime = 0;
 int16_t ax, ay, az;
 float Volt, Temperatur, Tilt, Gravity;
 
-float scaleTemperature(float t)
+float scaleTemperatureFromC(float t, uint8_t tempscale)
 {
-  if (myData.my_tempscale == TEMP_CELSIUS)
+  if (tempscale == TEMP_CELSIUS)
```

Comment:
Why do not use switch here?

Reply:
Could do, I wasn't intending to make great swathes of changes all at once.